stdlib/universe/parallel.go

```diff
@@ -0,0 +1,168 @@
+package universe
+
+import (
+	"context"
+	"fmt"
+	"math"
+	"sync"
+
+	"github.com/influxdata/flux"
+	"github.com/influxdata/flux/execute"
+	"github.com/influxdata/flux/execute/table"
+	"github.com/influxdata/flux/memory"
+	"github.com/influxdata/flux/plan"
+	"github.com/opentracing/opentracing-go"
+)
+
+const (
+	PartitionMergeKind = "influx2x/partitionMergeKind"
```

Comment:
Should this be called `ParallelMergeKind` instead? Since a partition is a storage concept. Or maybe you are introducing it as a Flux concept as well right now. In any case could you add some comments here describing what this node does?

Reply:
Ah yes. This was originally in IDPE and I was mostly using "partition" instead of "parallel". Will fix.